main/res/layout/coordinatescalculate_dialog.xml

```diff
@@ -48,8 +47,7 @@
 
                 android:paddingRight="16dp"
                 android:gravity="end"
-                android:textAlignment="gravity"
-                android:text="@string/lon_point_min_format"/>
```

Comment:
What was the purpose of this text previously? I am just curious.

Reply:
Oh, only just saw this.

To be honest I'm not exactly sure.  It's "default" text that's being placed in the results field but it never gets seen.  It's never seen by the user because as soon as the calculator opens this test gets replaced with the results of the calculation itself:
![image](https://user-images.githubusercontent.com/6573522/32107926-973a9c48-bb77-11e7-9009-5242ee2158c5.png)

I don't recall why a gave it a default value but it probably had something to do with my wanting to see what the layout looked like before the actual result code was working.  A bit like how people put "Lorem Ipsum" text in mock layouts to get an idea oh how things look.